cmd/juno/juno.go

```diff
@@ -221,6 +255,12 @@ func NewCmd(config *node.Config, run func(*cobra.Command, []string) error) *cobr
 	junoCmd.Flags().Uint16(wsPortF, defaultWSPort, wsPortUsage)
 	junoCmd.Flags().String(dbPathF, defaultDBPath, dbPathUsage)
 	junoCmd.Flags().Var(&defaultNetwork, networkF, networkUsage)
+	junoCmd.Flags().String(feederURLF, defaultFeederURL, networkCustomFeederUsage)
+	junoCmd.Flags().String(gatewayURLF, defaultGatewayURL, networkCustomGatewayUsage)
+	junoCmd.Flags().String(chainIDF, defaultChainID, networkCustomL2ChainIDUsage)
+	junoCmd.Flags().Int(l1ChainIDF, defaultL1ChainIDInt, networkCustomL1ChainIDUsage)
+	junoCmd.Flags().Int(protocolIDF, defaultProtocolID, networkCustomProtocolIDUsage)
+	junoCmd.Flags().String(coreContractAddressF, defaultCoreContractAddressStr, networkCustomCoreContractAddressUsage)
```

Comment:
do you think we can prefix these with some string to indicate that they are related?

something like;

`--cn.feeder-url`
`--cn.gateway-url`

Reply:
https://github.com/NethermindEth/juno/pull/1542/files#diff-e8011febe1f0287e6ed8aba108ce6e448c8db2c49f5afef912db1626eafd0ee3R70-R75

I don't see this change

Reply:
Whoops! Forgot to prefix the actual flags. Should be present now